### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by xitonix**

### CODE REVIEW

**Review by xitonix — COMMENTED** [review]
sdfsdf

**Comment by sandbox-alg** [comment]
sdfsdf

**Review by sandbox-alg — COMMENTED** [review]
sdfsdf

**Review by sandbox-alg — APPROVED** [review]

**Review by sandbox-alg — CHANGES_REQUESTED** [review]
sdfsdf

**Comment by xitonix** [comment]
asdasdasd

File: README.md [hunk 1]
```diff
@@ -29,6 +29,3 @@ Once you installed the tool, you can simply use `ssh` or `scp` the normal way.
 ssh -i "identity.pem" user@host
 scp -i "identity.pem" user@host:/src/file /dest/file
 ```
```
Comment (xitonix) [inline README.md:31]: werwerwe

Comment (xitonix) [reply README.md:31]: werwer

Comment (sandbox-alg) [reply README.md:31]: werwer

**Review by xitonix — COMMENTED** [review]
werwersdfsdf

File: README.md (hunk 1 above)
Comment (xitonix) [reply README.md:31]: asdasd

Comment (xitonix) [reply README.md:31]: asdasda

Comment (xitonix) [reply README.md:31]: sdfsdf

Comment (xitonix) [reply README.md:31]: asdasd

**Review by xitonix — COMMENTED** [review]
sdfsdf

**Review by xitonix — COMMENTED** [review]
asdasd

**Pull request closed without merging**